RSSReader/View/WebViewController.m

```diff
@@ -0,0 +1,172 @@
+//
+//  WebViewController.m
+//  RSSReader
+//
+//  Created by Uladzislau Stankevich on 7.01.21.
+//
+
+#import "WebViewController.h"
+#import <WebKit/WebKit.h>
+
+@interface WebViewController () <UIScrollViewDelegate>
+
+@property (nonatomic, assign)WKWebView *webView;
+@property (nonatomic, retain)UIBarButtonItem *backBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *forwardBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *refreshBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *stopBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *safariBarButtonItem;
+@property (nonatomic, retain)UIBarButtonItem *spacerBarButtonItem;
+@property (nonatomic) CGFloat contentOffsetLastY;
+
+@end
+
+@implementation WebViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.view.backgroundColor = UIColor.whiteColor;
```

Comment:
It would be better to use same color scheme across the application. Background color should be added to the color assets.

Reply:
fixed